dll/win32/uxtheme/nonclient.c

```diff
@@ -1266,10 +1299,17 @@ HRESULT WINAPI DrawNCPreview(HDC hDC,
         SelectFont(hDC, textFont);
 
     SetTextColor(hDC, GetThemeSysColor(context.theme, TMT_WINDOWTEXT));
-    DrawThemeText(context.theme, hDC, WP_DIALOG, 0, L"Window Text", -1, DT_LEFT | DT_TOP | textDrawFlags, 0, &rcWindowClient);
+    len = LoadStringW(hDllInst, IDS_WINTEXT, (LPWSTR)&szText, 0);
+    if (len > 0)
+        DrawThemeText(context.theme, hDC, WP_DIALOG, 0, szText, len, DT_LEFT | DT_TOP | textDrawFlags, 0, &rcWindowClient);
 
     /* Draw preview dialog window */
-    SetWindowTextW(hwndDummy, L"Message Box");
+    szText = LoadResourceString(hDllInst, IDS_MESSAGEBOX);
```

Comment:
hmm, why are you sometimes using LoadStringW, and some other times this LoadResourceString function?

Reply:
Thinking about your question, I've simplified the code a bit now. 🙂